caluma/caluma_workflow/tests/test_reopen_case.py

```diff
@@ -0,0 +1,214 @@
+import pytest
+from django.core.exceptions import ValidationError
+from graphene_django.utils.str_converters import to_const
+
+from .. import api, models
+
+
+def reopen_case_setup(case_status, work_item_statuses, case_factory, work_item_factory):
```

Comment:
You can turn this into a fixture using the `@pytest.fixture` decorator, returning a function to do the actual work. This would save you some typing on the call sites.